bin/install.sh

```diff
@@ -200,10 +197,6 @@ function run_ansible_playbook() {
 function upgrade_docker_containers() {
     display_section "Initialize/Upgrade Docker Containers"
 
-    wget -q \
-        "$GITHUB_RAW_URL/master/bin/upgrade_containers.sh" \
-        -O "$UPGRADE_SCRIPT_PATH"
```

Comment:
Why was this removed?
(What are the issues that you've encountered before removing this?)

Reply:
This was a mistake on my part. I've removed this change from my code and it should no longer be a problem.